Cannot import appengine

### How do you use Sentry?

Sentry Saas (sentry.io)

### Version

1.18.0

### Steps to Reproduce

Install the SDK within any project that is not pinning urllib3 < 2.0.0

### Expected Result

ability to import appengine

### Actual Result

Cannot import appengine as gaecontrib.
As per urllib 2.0.0 release: https://github.com/urllib3/urllib3/tree/2.0.0

Removed urllib3.contrib.appengine.AppEngineManager and support for Google App Engine Standard Environment (https://github.com/urllib3/urllib3/issues/2044).

In AWS Lamdba I got 

```
[ERROR] Runtime.ImportModuleError: Unable to import module 'step_function_trigger': urllib3 v2.0 only supports OpenSSL 1.1.1+, currently the 'ssl' module is compiled with OpenSSL 1.0.2k-fips  26 Jan 2017. See: https://github.com/urllib3/urllib3/issues/2168
Traceback (most recent call last):
```

--- a/setup.py
+++ b/setup.py
@@ -41,6 +41,7 @@ setup(
         'urllib3>=1.25.7; python_version<="3.4"',
         'urllib3>=1.26.9; python_version=="3.5"',
         'urllib3>=1.26.11; python_version >="3.6"',
+        'urllib3<2.0.0',
         "certifi",
     ],
     extras_require={

